src/components/widgets-settings/organizations/organizations.directive.coffee

```diff
@@ -1,15 +1,28 @@
 module = angular.module('impac.components.widgets-settings.organizations',[])
-module.controller('SettingOrganizationsCtrl', ($scope, $log, ImpacDashboardsSvc) ->
+module.controller('SettingOrganizationsCtrl', ($scope, $log, ImpacDashboardsSvc, ImpacMainSvc) ->
 
   w = $scope.parentWidget
   w.selectedOrganizations = {}
 
+
+  $scope.organizationMode ||= 'multiple'
```

Comment:
@xaun  Currently I've have added the ability to use `single` mode but yes it's not being used anywhere. As per discussion with @cesar-tonnoir , this is something the developers will activate when coding a specific widget.

I've also added a message in PR description on how it can be used for a widget.